src/components/templates/PartyMap/components/RoomModal/RoomModal.scss

```diff
@@ -45,6 +38,17 @@ $spacing: 20px;
     }
   }
 
+  &__icon {
+    flex-shrink: 0;
+    height: 180px;
+    width: 180px;
+    border-radius: $border-radius;
+    background-color: transparent;
```

Comment:
I believe @mcflyok's suggestion was to use a different explicit colour for this rather than `transparent`?

- https://github.com/sparkletown/sparkle/pull/1272#issuecomment-809135362
  - > I’d go for contain+center to show the full image contained inside the same size square, **with the background color** and border radius, as I believe most of our room images will be transparent pngs
  
I'm not sure what the best color to use here is (@mcflyok thoughts?), but perhaps `$black` is fine?

Reply:
his designs use black so I'll switch it to that